kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/kview.css

```diff
@@ -4092,3 +4092,117 @@ Styling a context menu for kview ui/ux controls
     -fx-background-insets: 0;
     -fx-background-radius: 6;
 }
+
+/*****************************
+* next gen reasoner styles
+*****************************/
+.grey1-label {
+    -fx-text-fill: -Grey-1;
```

Comment:
Can you make this more generic?
Maybe the following:
```css
 /* result label */
.reasoner-result-container .label {
   -fx-text-fill: -Grey-1;
   -fx-font-family: "Noto Sans";
   -fx-font-size: 14px;
}
```

Reply:
fixed